src/compilerWrapper.ts

```diff
@@ -368,4 +370,47 @@ function getABIDeclaration(astRoot): { contract: string, abi: Array<ABIEntity> }
 		contract: mainContract['name'],
 		abi: interfaces
 	};
-}
\ No newline at end of file
+}
+
+export function getPlatformsCryptc() : string {
+	switch (os.platform()) {
+		case "win32":
+			return "compiler/scryptc/win32/scryptc.exe";
+		case "linux":
+			return "compiler/scryptc/linux/scryptc";
+		case "darwin":
+			return "compiler/scryptc/mac/scryptc";
+		default:
+			throw "sCrypt don't support your OS now";
+	}
+}
+
+
+
+export function getDefaultsCryptc(): string {
+	const homedir = os.homedir();
+	const extensions =  join(homedir, ".vscode/extensions");
+	if(!existsSync(extensions)) {
+		throw `No vscode extensions found, Please check if vscode is installed on your machine.`
+	}
+
+    let sCrypt = readdirSync(extensions).filter(dir => {
+		if(dir.indexOf("bsv-scrypt.scrypt-") > -1 ) {
+			return true;
+		} 
+		return false;
+	});
+
+	if(sCrypt.length == 0) {
+		throw `No sCrypt IDE found, Please install sCrypt IDE at vscode extensions marketplace: https://marketplace.visualstudio.com/items?itemName=bsv-scrypt.sCrypt`;
+	} 
+
+	let scryptc = join(extensions, sCrypt[0], getPlatformsCryptc());
+
+	if(!existsSync(scryptc)) {
+		throw `No scryptc found, Please update your sCrypt extension to latest version`;
```

Comment:
throw `No sCrypt compiler found. Please update your sCrypt extension to the latest version`;